settings.py

```diff
@@ -151,7 +151,8 @@
 SESSION_ENGINE = "django.contrib.sessions.backends.cached_db"
 
 # time in minutes before forced logout due to inactivity
-INACTIVITY_TIMEOUT = 30
+INACTIVITY_TIMEOUT = 60 * 24 * 14
```

Comment:
I am not attached to the current setup, and would be happy to change if you think its better.
I put it in settings because I feel like this is more configuration than domain setting. Domains can choose if they want short or long, but what those values mean should probably be server wide configurations and up to us not the domains. I think it also simplifies the logic because we can store a boolean on the domain instead of converting the checkbox status to a number, and we never need to worry about comparing domain values to current session values to find the smaller one. Once the session is secure, we dont care what the domain says, and until then, we only check if the domain wants secure sessions not how there unique number compares to the current session value.